### DISCUSSION TITLE
I ask refund.Tried on business udemy doesnt work.

### DISCUSSION THREAD

**Discussion by izguovdq7020** [aclap-dev/video-downloadhelper#1234] [License & Purchase requests]
### What's your issue?

Email:izgdum123@gmail.com Purchased 2 hours ag

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by izguovdq7020** [comment]
e518abc63f17467

**Comment by jcfrog** [comment]
@izguovdq7020 Hi,
You have been refunded. Should be available in a few business days.
